models/bpreg/config/default.yml

```diff
@@ -0,0 +1,14 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: Body part regression pipeline from DKFZ, processing DICOM files to predict anatomical regions and anomalies
+execute:
+- DicomImporter
+- NiftiConverter
+- BpRegRunner
+- DataOrganizer
+
+modules:
+  DataOrganizer:
+    targets:
+      - json:type=utility-->[i:sid]/utility.json
```

Comment:
The main output file should ideally be named after the model (`bpreg.json` in this case then) to avoid any clashes when running multiple MHub models on the same input data into the same output structure.